### PR TITLE
[polish]bloom_int8_tp.py  complete_compare

### PR DESCRIPTION
**Pull request by ht-zhou**

### CODE REVIEW

File: bloom-inference-scripts/bloom_int8_tp.py [hunk 1]
```diff
@@ -127,39 +128,67 @@ def run_tp():
...
+    outputs2 = model2(**inputs, labels=inputs["input_ids"])
+    output2 = outputs2.logits
+
+    model3 = BloomForCausalLM.from_pretrained("/data2/users/lczht/bloom-560m").to(rank)
+    outputs3 = model3(**inputs, labels=inputs["input_ids"])
+    output3 = outputs3.logits
+
+    print(torch.max(abs(output - output2)))
+    print(torch.max(abs(output - output3)))
+    print(torch.max(abs(output2 - output3)))
```
Comment (feifeibear) [inline bloom-inference-scripts/bloom_int8_tp.py:189]: assert 它们小于一个值比如1e-6

Comment (feifeibear) [reply bloom-inference-scripts/bloom_int8_tp.py:189]: 用torch.allclose

File: bloom-inference-scripts/bloom_int8_tp.py [hunk 2]
```diff
@@ -127,39 +128,67 @@ def run_tp():
...
+    
+    print(logits)
     
-    outputs = model.generate(**inputs, labels=inputs["input_ids"])
-    output = tokenizer.batch_decode(outputs, skip_special_tokens=True)
-    print(outputs)
-    print(output)
+    # outputs = model.generate(**inputs, labels=inputs["input_ids"])
+    # output = tokenizer.batch_decode(outputs, skip_special_tokens=True)
+    # print(output)
```
[the remark this answers is not in the document]
Comment (ht-zhou) [reply bloom-inference-scripts/bloom_int8_tp.py]: ok

File: bloom-inference-scripts/bloom_int8_tp.py (hunk 1 above)
Comment (ht-zhou) [reply bloom-inference-scripts/bloom_int8_tp.py]: ok

**Review by feifeibear — APPROVED** [review]

**Pull request merged**